client/src/redux/actions/userAction.js

```diff
@@ -20,10 +20,10 @@ export const userRegister = (username, email, password) => async (dispatch) => {
     const config = {
       "Content-Type": "application/json",
     };
-    const baseUrl="https://mydiary-backend-host.herokuapp.com";
+    const baseUrl = "https://mydiary-backend-host.herokuapp.com";
```

Comment:
I'll prefer this to be used as a global variable.

Create a const.js add these baseurl there